perllib/Open311/PopulateServiceList.pm

```diff
@@ -335,27 +343,6 @@ sub _delete_contacts_not_in_service_list {
 sub _delete_contacts_not_in_service_list_cobrand_overrides {
     my ( $self, $found_contacts ) = @_;
 
-    # for Warwickshire/Bristol/BANES, which are mixed Open311 and email, don't delete
-    # the email addresses
-    if ($self->_current_body->name eq 'Warwickshire County Council' ||
-        $self->_current_body->name eq 'Bristol City Council' ||
-        $self->_current_body->name eq 'Bath and North East Somerset Council') {
-        $found_contacts = $found_contacts->search(
-            {
-                email => { -not_like => '%@%' }
-            }
-        );
-    } elsif ($self->_current_body->name eq 'East Hertfordshire District Council' ||
```

Comment:
Is it ok to remove this?

Reply:
Yeah, the Stevenage/east Herts "Other" category is also an email, so should be caught by the same thing.